pages/common/pueue-add.md

```diff
@@ -0,0 +1,28 @@
+# pueue add
+
+> Enqueue a task for execution.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Add any command to the default queue:
+
```

Comment:
I think we should remove both this and the next example and replace them with a combination, something like this:
```markdown
- Add a command to the queue in the default group:

`pueue add -- {{command}} {{args}}`
```